container/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
```

Comment:
Is this file copied from somewhere? If so, it wouldn't hurt to link to the original source in a comment for both credit and knowing where to find updates in the future.

Reply:
Yes, I did ~steal~ copy it from somewhere. I'll add a comment explaining its provenance.